src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -15,11 +15,10 @@ public MyHashMap() {
     @Override
     public void put(K key, V value) {
         checkCapacity();
-        int position = getPosition(key);
-        Bucket<K, V> bucket = table[position];
+        Bucket<K, V> bucket = table[getPosition(key)];
         Bucket<K, V> newBucket = new Bucket<>(key, value);
-        if (table[position] == null) {
-            table[position] = newBucket;
+        if (table[getPosition(key)] == null) {
+            table[getPosition(key)] = newBucket;
```

Comment:
In this piece of code introducing variable int position is argumented, because it is used serveral times. but ok, we can avoid it usage.